apps/console/src/features/branding/components/design/design-form.tsx

```diff
@@ -2130,109 +2130,6 @@ export const DesignForm: FunctionComponent<DesignFormPropsInterface> = forwardRe
                 </Grid.Column>
             </Grid.Row>
 
-            <Grid.Row columns={ 1 }>
```

Comment:
No need to remove the secondary button color config.
Still this is valid for screens like SMS OTP, etc.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4544/commits/6f3b62f11f799db0f0bc5a9eec851e541e719e72